pkg/constants/constants.go

```diff
@@ -30,6 +30,9 @@ const (
 	// MetricsPort is the port for the prometheus metrics endpoint.
 	MetricsPort = 7777
 
+	// ConnectRetries is the number of retries for an implicit route.
+	ConnectRetries = 10
```

Comment:
Does this reconcile with the default in the NATS server?

Reply:
This is higher than the default from the NATS server, which is only a couple of times.  It takes sometime in Kubernetes for DNS service to be ready so sometimes routes would have been dropped